user_stories.md

```diff
@@ -0,0 +1,46 @@
+### User Stories
+
+## User Story 1
```

Comment:
great to see all the user stories in this file.  really helped to follow each story with each test